internal/waterloo-api-connector/waterloo-api-connector.py

```diff
@@ -0,0 +1,36 @@
+import requests
```

Comment:
can you move this python script out of the `internal/` folder into a new `scripts/` folder so we don't mix our scripts and backend stuff together? i would also call it something more specific to fetching courses and not just a "connector"

Reply:
I was thinking we could put our backend under `internal/apps/course-planner/...` and the python app `internal/apps/waterloo-api-connector/...`, but it's a lot of refactoring

Reply:
I told Kevin to name it a connector because I viewed it as more of another app rather than just a script, but we can treat it like a script too

Reply:
i think it makes more sense to think of it a script, only need to run it when something changes in UW backend (compared to something we run regularly)